graph.go

```diff
@@ -95,7 +95,9 @@ type NodeRemover interface {
 // EdgeSetter is an interface for adding edges to a graph.
 type EdgeSetter interface {
 	// SetEdge adds an edge from one node to another.
-	// If the nodes do not exist, they are added.
+	// If the graph supports node addition the nodes
+	// will be added if they do not exist, otherwise
+	// SetEdge will panic.
 	// SetEdge will panic if the IDs of the e.From
```

Comment:
It is clear to what `e` refers but strictly speaking there is no `e` around. Could you please reword this somehow?